src/main/scala/org/zalando/react/nakadi/client/providers/Events.scala

```diff
@@ -217,10 +220,10 @@ class ProduceEvents(properties: ProducerProperties,
       Source
         .single(finalRequest)
         .via(clientProvider.connection)
-        .runWith(Sink.foreach {
-          case HttpResponse(status, _, _, _) if status.isSuccess() => ()
+        .mapAsync(4) {
```

Comment:
Good point. We can leave as is. 

I think some stage it would be good to gather all these configuration options across areas where concurrency is managed, and make it configurable. For now we'll leave as is.